nntrainer/tensor/blas_interface.cpp

```diff
@@ -847,10 +847,10 @@ unsigned int isamax(const unsigned int N, const float *X, const int incX) {
 #endif
 }
 
-void sine_transformation(const unsigned int N, float *X, float *Y,
+void sine(const unsigned int N, float *X, float *Y,
                          float alpha) {
 #ifdef USE_NEON
-  nntrainer::neon::sine_transformation_neon(N, X, Y, alpha);
+  nntrainer::neon::sine_neon(N, X, Y, alpha);
```

Comment:
If you look at the namespace, neon already exists.
Can't I just use sine or sin?

Reply:
With this logic, every function in `blas_neon.cpp` must be revised.
Will do it in the near future.